src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -76,9 +76,16 @@
     <Compile Include="$(CommonPath)\System\IO\PathInternal.CaseSensitivity.cs">
       <Link>Common\System\IO\PathInternal.CaseSensitivity.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\Threading\Tasks\TaskToApm.cs">
+      <Link>Common\System\Threading\Tasks\TaskToApm.cs</Link>
```

Comment:
><Link>Common\System\Threading\Tasks\TaskToApm.cs</Link> [](start = 6, length = 55)

In theory you will only need this for netstandard1.7, so why not add a condition to only compile when `'$(TargetGroup)'==''`